CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+## Breaking Changes:
+
+* Feat: Support envelope based transport for events (#391)
+* Feature: Envelope Only Transport API #426 (#463)
+  * The method signature of `Transport` changed from `Future<SentryId> send(SentryEvent event)` to `Future<SentryId> send(SentryEnvelope envelope)`
```

Comment:
we can add only the id of this PR now as it has everything.
I'd add an entry for Bump Ccocoa 7.1.3